src/errors/index.js

```diff
@@ -1,42 +1,2 @@
-function addDevError(acc, code, message) {
-  acc[code] = `${code}: ${message}
-
-    You can find more information here: https://github.com/obartra/reflex/wiki/${code}`
-  return acc
-}
-
-function addProdError(acc, code) {
-  acc[code] = code
-  return acc
-}
-const addError =
-  process.env.NODE_ENV === 'production' ? addProdError : addDevError
-const errors = {}
-
-addError(
-  errors,
-  'INVALIDMEDIAKEY',
-  `Specified query is invalid. Only the following keys are allowed: "minWidth", "maxWidth", "minHeight", "maxHeight", "aspectRatio" and "orientation".`
-)
-addError(
-  errors,
-  'INVALIDSPACING',
-  `Invalid spacing property type used, only array, undefined, string or numbers allowed.`
-)
-addError(
-  errors,
-  'MIXEDSPACING',
-  `You cannot define margin or padding and a direction at the same time.`
-)
-addError(
-  errors,
-  'NOMATCHMEDIA',
-  `"window.matchMedia" is not available in your environment, media queries will not work.`
-)
-addError(
-  errors,
-  'TOOMANYSPACEVAL',
-  `Invalid Spacing size, only first 4 values used. Spacing values follow CSS syntax, this means you can specify 1, 2, 3 or 4 values but not more.`
-)
-
-module.exports = errors
+// @preval
+module.exports = require('../errors')
```

Comment:
I moved errors to `./errors/errors.js` and the `./errors/index.js` simply preval's it. This simplifies importing errors (no need to be concerned about `preval`, you can import with `import errors from './errors'`)